From 1772a92531f9198b2e10f0bf2cf5c7a13435c5ac Mon Sep 17 00:00:00 2001
From: Gabriel Craciunescu <nix.or.die@gmail.com>
Date: Mon, 5 Nov 2018 16:16:31 +0100
Subject: [PATCH] CreatePartitionTableJob: drop lsblk and mount debug output

 is done wrong anyway .. not really needed ( and like this not needed et all )
---
 .../jobs/CreatePartitionTableJob.cpp          | 20 -------------------
 1 file changed, 20 deletions(-)

diff --git a/src/modules/partition/jobs/CreatePartitionTableJob.cpp b/src/modules/partition/jobs/CreatePartitionTableJob.cpp
index 937b8437d..8719ed389 100644
--- a/src/modules/partition/jobs/CreatePartitionTableJob.cpp
+++ b/src/modules/partition/jobs/CreatePartitionTableJob.cpp
@@ -85,26 +85,6 @@ CreatePartitionTableJob::exec()
     cDebug() << "Creating new partition table of type" << table->typeName()
              << ", uncommitted yet:";
 
-    if ( Logger::logLevelEnabled( Logger::LOGDEBUG ) )
-    {
-        for ( auto it = PartitionIterator::begin( table );
-            it != PartitionIterator::end( table ); ++it )
-            cDebug() << *it;
-
-        QProcess lsblk;
-        lsblk.setProgram( "lsblk" );
-        lsblk.setProcessChannelMode( QProcess::MergedChannels );
-        lsblk.start();
-        lsblk.waitForFinished();
-        cDebug() << "lsblk:\n" << lsblk.readAllStandardOutput();
-
-        QProcess mount;
-        mount.setProgram( "mount" );
-        mount.setProcessChannelMode( QProcess::MergedChannels );
-        mount.start();
-        mount.waitForFinished();
-        cDebug() << "mount:\n" << mount.readAllStandardOutput();
-    }
 
     CreatePartitionTableOperation op(*m_device, table);
     op.setStatus(Operation::StatusRunning);
-- 
2.19.1

